go/apps/api/openapi/spec/error/ValidationError.yaml

```
@@ -0,0 +1,34 @@
+      present, they offer specific remediation advice.
+    type: string
+    example: Ensure the name uses only alphanumeric characters, underscores,
+      and hyphens
+required:
+  - location
+  - message
+type: object
+description:
+  Individual validation error details. Each validation error provides
+  precise information about what failed, where it failed, and how to fix it,
+  enabling efficient error resolution.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)